TheMovieDB/TheMovieDB/View/Cells/CollectionViewCell.swift

```diff
@@ -23,16 +23,23 @@ class CollectionViewCell: UICollectionViewCell {
     
     override init(frame: CGRect) {
         super.init(frame: frame)
+        setUp()
+    }
+    
+    required init?(coder: NSCoder) {
+        return nil
+    }
+    
+    func setUp() {
         contentView.backgroundColor = .black
         contentView.layer.cornerRadius = 10
         contentView.clipsToBounds = true
+        titleLayout()
```

Comment:
Add the set word so these methods are more descriptive like setTitleLayout, etc.